app/views/schools/low_carbon_hub_installations/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Low carbon hub installation <%= @low_carbon_hub_installation.rbee_meter_id %></h1>
+<h2><%= @school.name %></h2>
+
+<p><%= link_to "All low carbon hub installations for #{@school.name}",  school_low_carbon_hub_installations_path(@school) %>
+<% JSON.parse(@low_carbon_hub_installation.information).each do |key, value| %>
```

Comment:
Do these need to be in a `<dl>`?

Reply:
Not only that, but I've got the dt and the dd the wrong way around - just been working with HTML since 1995, no biggie